loader/loader.go

```diff
@@ -55,7 +55,13 @@ func (l *Result) Compiler() (*ast.Compiler, error) {
 
 // Store returns a Store object with the documents from this loader result.
 func (l *Result) Store() (storage.Store, error) {
```

Comment:
I couldn't find this one being used in our code bases, but I figured for completeness' sake, we could still add 👇